CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ## [Unreleased]
 ### Added
 - Command Generate - Add support for MySQL 8.
+- Command Update - New command update insert new tables/fields/relationships.
```

Comment:
```suggestion
- Command Update - New update command to insert new tables/fields/relationships.
```

Reply:
This is an interesting topic for sure.
I would have created a new topic to discuss about that in [Innovation-ideas](https://github.com/ForestAdmin/innovation-ideas/issues)

You do not provide any information/description about how it behaves in the PR. You know everything because you wrote it. Really hard for the reviewer to understand the context and what you have in mind for this feature.